### PR TITLE
Add: Some logging for JavaCmd Class

### PR DESCRIPTION
**Pull request by y0urself**
## What

Add logging to specific functions

## Why

For debugging I need some more information, that can be achieved with logging, best

## Checklist

- [ ] Tests

### CODE REVIEW

File: pontos/release/main.py [hunk 1]
```diff
@@ -32,6 +33,8 @@ def main(
     args=None,
 ) -> NoReturn:
     username, token, parsed_args = parse_args(args)
+    logging.basicConfig(format="%(levelname)s - %(name)s - %(message)s")
```
Comment (bjoernricks) [inline pontos/release/main.py:36]: Can we disable logging if `--quiet` is passed?

**Review by bjoernricks — COMMENTED** [review]
Also please fill in the PR template

File: pontos/release/main.py (hunk 1 above)
Comment (y0urself) [reply pontos/release/main.py:36]: I am not sure if I can implement this.
I would need to pass the quiet argument to the `VersionCommand` class, I could also pass the `RichTerminal` to that class in this case and wouldn't need to use `logging` at all?
My idea was to not add any passed arguments, thus I wanted to go with `logging` ... you have any idea?

Comment (bjoernricks) [reply pontos/release/main.py:36]: You can just disable all logging globally https://docs.python.org/3/library/logging.html#logging.disable

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**